build.gradle.kts

```diff
@@ -190,17 +191,6 @@ tasks {
         dependsOn("integration")
     }
 
-    register("bootRunLocal") {
```

Comment:
why we remeove runLLocal

Reply:
because email-sender does not have db so local profile works like default so running bootRunLocal == running bootRun